Cargo.toml

```diff
@@ -15,5 +15,13 @@ include = [ "**/*.rs", "Cargo.toml"]
 [dependencies]
 sdl2 = "0.34.3"
 gl = "0.14.0"
-clipboard = "0.5"
-egui = "0.10.0"
\ No newline at end of file
+egui = "0.10.0"
+
+[dependencies.clipboard]
+package = "cli-clipboard"
+#version = "0.5"
+version = "0.2"
+optional = true
+
+[features]
+default = ["clipboard"]
```

Comment:
I'm not very familiar with the features concept of cargo, so you will have to help me understand your changes. I gather here that you've enabled clipboard as a default feature? If someone doesn't want clipboard they have to compile with default features turned off. Is that it?

Reply:
Yes.
To disable clipboard support you can simply set `defaut-features = false` for dependency in Cargo.toml
I set it on by default because such usage is very rare in practice.